src/main/java/cafeboard/Board/BoardService.java

```diff
@@ -0,0 +1,59 @@
+package cafeboard.Board;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+public class BoardService {
+
+    private final BoardRepository boardRepository;
+
+    public BoardService(BoardRepository boardRepository) {
+        this.boardRepository = boardRepository;
+    }
+
+    //board 생성
+    public BoardResponse create (BoardRequest request){
+        Board board = new Board(request.title());
+
+                boardRepository.save(board);
+
+        return new BoardResponse(board.getBoardId(),
+                                board.getTitle());
+    }
+
+    public List<BoardResponse> findAll (){
+        return boardRepository.findAll()
+                .stream()
+                .map(board -> new BoardResponse(
+                        board.getBoardId(),
+                        board.getTitle()))
+                .toList();
+    }
+
+    //게시판 수정
+    @Transactional
+    public BoardResponse update(Long boardId, BoardRequest request){
+        Board board = boardRepository.findById(boardId).orElseThrow(
+                () -> new IllegalArgumentException("수정할 게시판이 존재하지 않습니다.")
+        );
+
+        board.setTitle(request.title());
+
+        return new BoardResponse(board.getBoardId(),
+                board.getTitle());
+    }
+
+    //게시판 조회
+    @Transactional
+    public void delete(Long boardId){
```

Comment:
사소한 실수이지만, 주석을 달면 그 주석조차도 유지보수의 대상이 된다는 사실
```suggestion
    //게시판 삭제
    @Transactional
    public void delete(Long boardId){
```